### PR TITLE
Added configuration to attach external debugger

### PR DESCRIPTION
**Pull request by Emulebest**

### CODE REVIEW

File: papiea-engine/docker-compose-debug.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+                ports:
+                        - "3333:3000"
+                        - "9229:9229"
+                environment:
+                        - ADMIN_S2S_KEY=${PAPIEA_ADMIN_S2S_KEY}
+                        - PAPIEA_PUBLIC_ADDR=${PAPIEA_PUBLIC_ADDR}
+                        - HOT_RELOAD=true
+                        - PAPIEA_DEBUG=true
+        mongo:
+                image: mongo:4.1
```
Comment (adolgarev) [inline papiea-engine/docker-compose-debug.yml]:
```
papieajs$ diff -u papiea-engine/docker-compose.yml papiea-engine/docker-compose-debug.yml
--- papiea-engine/docker-compose.yml    2019-06-25 14:33:43.060761360 +0300
+++ papiea-engine/docker-compose-debug.yml      2019-07-10 19:11:07.869290886 +0300
@@ -10,9 +10,11 @@
                         - mongo
                 ports:
                         - "3333:3000"
+                        - "9229:9229"
                 environment:
                         - ADMIN_S2S_KEY=${PAPIEA_ADMIN_S2S_KEY}
                         - PAPIEA_PUBLIC_ADDR=${PAPIEA_PUBLIC_ADDR}
-                        - HOT_RELOAD=${HOT_RELOAD:-false}
+                        - HOT_RELOAD=true
+                        - PAPIEA_DEBUG=true
         mongo:
                 image: mongo:4.1
```
I believe we can use one papiea-engine/docker-compose.yml with HOT_RELOAD and PAPIEA_DEBUG set from env vars

Comment (shlomi-nx) [reply papiea-engine/docker-compose-debug.yml]: Yes, it may be easier to combine docker-compose files if possible.

Comment (Emulebest) [reply papiea-engine/docker-compose-debug.yml]: I am a little worried about exposing additional ports

Comment (shlomi-nx) [reply papiea-engine/docker-compose-debug.yml]: I see your point. Lets keep this separate for now.

**Pull request merged**